reo/src/pvwatts.py

```diff
@@ -48,7 +48,8 @@ def check_pvwatts_response_data(resp):
         message = ("Error parsing PVWatts response. "
                    "Status code: {}. Text: {}".format(resp.status_code, resp.text))
         raise_pvwatts_exception(message)
-    return data
+    else:
```

Comment:
Just curious, why was this change needed? If an exception is encountered we'll exit out of this workflow through the raise funciton

Reply:
Two reasons: 
1. clarity: the `else` statement says if no exception occurs then do this
2. robustness: if the `except` block is ever changed to something that does not raise an exception then the code would fail (since `data` would not be defined due to the scoping of the try/except block)